Externals/SlippiRustExtensions/jukebox/src/lib.rs

```diff
@@ -103,13 +107,40 @@ impl Jukebox {
         let (message_dispatcher_thread_tx, message_dispatcher_thread_rx) = channel::<JukeboxEvent>();
         let (music_thread_tx, music_thread_rx) = channel::<JukeboxEvent>();
 
+        // Spawn message dispatcher thread
         std::thread::Builder::new()
             .name("JukeboxMessageDispatcher".to_string())
-            .spawn(move || Self::dispatch_messages(m_p_ram, get_dolphin_volume, message_dispatcher_thread_rx, melee_event_tx))?;
+            .spawn(move || {
+                match Self::dispatch_messages(m_p_ram, get_dolphin_volume, message_dispatcher_thread_rx, melee_event_tx) {
```

Comment:
This could be `if let Err(e) = Self::dispatch_message(...)` if you wanted to drop an indent level. :)

Reply:
These have the same indentation level, so I'll leave it as is for consistency with the other thread's error handler:
```rs
// Match 
match Self::dispatch_messages(m_p_ram, get_dolphin_volume, message_dispatcher_thread_rx, melee_event_tx) {
    Err(e) => tracing::error!(
        target: Log::Jukebox,
        error = ?e,
        "JukeboxMessageDispatcher thread encountered an error: {e}"
    ),
    _ => (),
}

// if-let
if let Err(e) = Self::dispatch_messages(m_p_ram, get_dolphin_volume, message_dispatcher_thread_rx, melee_event_tx) {
    tracing::error!(
        target: Log::Jukebox,
        error = ?e,
        "JukeboxMessageDispatcher thread encountered an error: {e}"
    );
}
```